### PR TITLE
[chores] Minor typographical error fix

### PR DESCRIPTION
**Pull request by devkapilbansal**
Related openwisp/openwisp-users#173

### CODE REVIEW

File: requirements-test.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 coveralls
 responses
 freezegun
-openwisp-users~=0.4.0
+openwisp-users~=0.4.1
```
Comment (pandafy) [inline requirements-test.txt:4]: This is unnecessary. Read [compatible release section of PEP 440](https://www.python.org/dev/peps/pep-0440/#compatible-release) specification to know more

File: requirements.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-openwisp-users~=0.4.0
+openwisp-users~=0.4.1
```
Comment (pandafy) [inline requirements.txt:1]: Same as above

**Review by pandafy — CHANGES_REQUESTED** [review]
Please don't make changes which are not related to the issue or PR

File: requirements.txt (hunk 2 above)
Comment (devkapilbansal) [reply requirements.txt:1]: Will update to v0.5 directly once it is released

**Review by nemesisdesign — APPROVED** [review]

**Pull request merged**